lib_xua/src/core/pdm_mics/mic_array_task.c

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022-2024 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#include "xua_conf.h"
+#if (XUA_NUM_PDM_MICS > 0)
+
+
+#include "xua_pdm_mic.h"
+#include "mic_array.h"
+
+#include <xcore/channel.h>
+
+
+#define CLRSR(c)                asm volatile("clrsr %0" : : "n"(c));
```

Comment:
This doen't appear to be used

Reply:
It is - in the next line

#define CLEAR_KEDI()            CLRSR(XS1_SR_KEDI_MASK)